docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -425,6 +426,7 @@ Create a container
           You must use this with `memory` and make the swap value larger than `memory`.
     -   **MemoryReservation** - Memory soft limit in bytes.
     -   **KernelMemory** - Kernel memory limit in bytes.
+    -   **NanoCPUs** - CPU quota in units of 10<sup>-9</sup> CPUs.
```

Comment:
Yes. API is an integer in units of 10<sup>-9</sup> CPUs. The CLI is a fraction number that represent the number of 1 CPUs (Could be partial). Created a PR #28075 for that. Please take a look.